Formula/kumitateru.rb

```diff
@@ -0,0 +1,21 @@
+# typed: false
+
+# frozen_string_literal: true
+
+# Kuitateru formula
```

Comment:
```suggestion
```
This isn't needed.

Reply:
Well, brew audit forced me do add this

Reply:
Then you ran a different audit than specified in the PR template. Likely `brew audit SomeFile.rb` instead of `brew audit formula`

Reply:
Where did you put `kumitateru.rb` when you ran it? This audit is enforced for certain directories inside `$(brew --repository)`, but won't be for files in `$(brew --repository homebrew/core)`.